src/components/categories.js

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { status } from '../redux/categories/categories';
 
-const Categories = () => (
-  <button type="button">Check Status</button>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+  return (
+    <button type="button" onClick={() => dispatch(status)}>Check Status</button>
+  );
+};
```

Comment:
- [ ] Kindly make sure that the check status button works as expected,  Onclick it should dispatch `under construction`

Reply:
Done :+1: